chart/otomi/localtest.sh

```diff
@@ -1,7 +1,8 @@
 set -e
 
-export VALUES_INPUT=/tmp/otomi/values.yaml
-# export CI=1
+export VALUES_INPUT=${VALUES_INPUT:-tmp/otomi/values.yaml}
+export ENV_DIR=${ENV_DIR:-/tmp/otomi/values}
```

Comment:
This makes no sense, and ENV_DIR is not a requirement for localist, so please remove